vitrina/orgs/templates/vitrina/orgs/list.html

```diff
@@ -54,13 +54,8 @@ <h1 class="title is-size-5-mobile">{% translate "Paieškos rezultatai" %}: {{ it
                                 <span class="icon is-medium is-hidden-mobile">
                                     <i class="fas fa-building"></i>
                                 </span>
-                                <strong><a href="/{{ radix }}/{{ i.slug }}" class="dataset-list-title">{{ i.title }}</a></strong>
+                                <strong><a href="/orgs/{{ i.kind }}/{{ i.slug }}" class="dataset-list-title">{{ i.title }}</a></strong>
```

Comment:
```suggestion
                                <strong><a href="{{ i.get_absolute_url }}" class="dataset-list-title">{{ i.title }}</a></strong>
```

Do not hardcode urls, instead reverse url by url name.

https://docs.djangoproject.com/en/3.2/topics/http/urls/#reverse-resolution-of-urls